frontend/src/lib/components/PropertiesTimeline/PropertiesTimeline.tsx

```diff
@@ -0,0 +1,45 @@
+import { Properties } from '@posthog/plugin-scaffold'
```

Comment:
Same as other `PropertiesTimeline`. Is this the right import?

Reply:
About `@posthog/plugin-scaffold`, basically yeah, since it's just a more semantic form of `Record<string, any>`, and inherently this form must be the same in both plugins and events returned from the API